apps/server/src/modules/school/service/school.service.spec.ts

```diff
@@ -36,8 +44,12 @@ describe('SchoolService', () => {
 	});
 
 	beforeEach(() => {
-		mockSchoolEntities = [schoolFactory.buildWithId(), schoolFactory.buildWithId()];
-		schoolDto = new SchoolDto({ name: 'schule1234' });
+		system = systemFactory.buildWithId();
+		const school1: School = schoolFactory.buildWithId({ name: 'schoolName' });
```

Comment:
Maybe: "school" and "schoolWithoutSystems"

Reply:
done